### PR TITLE
Update VSCode install/debug instructions.

### LINKED ISSUE #3257: Using Visual Studio Code for PowerShell Development - ext install instructions needs edits
Using Visual Studio Code for PowerShell Development
https://github.com/PowerShell/PowerShell/blob/master/docs/learning-powershell/using-vscode.md

This page lists the following:
•Press F1 (or Ctrl+Shift+P) which opens up the "Command Palette" inside the Visual Studio Code app.
• In the command palette, type ext install and hit Enter. It will show all Visual Studio Code extensions available on your system.

These instructions need some editing.  Simply typing ext install and hitting Enter selects the first option in the command palette list, which is to Disable All Installed Extensions

Mentioning @daviwil and @rkeithhill for their visibility

### PR DESCRIPTION
**Pull request by rkeithhill**
Fix #3257

### CODE REVIEW

File: docs/learning-powershell/using-vscode.md [hunk 1]
```diff
@@ -73,56 +73,91 @@ you will need to add a new variable to your user settings file.
...
--	To disable the breakpoint, press **F9** again.
--	Press **F5** to run the script.
-The execution should stop on the line you put the breakpoint on.
-- Press **F5** to continue running the script.
+This represents the common debug scenarios.
+However, when you open this file in the editor, you will see an **Add Configuration...** button.
+You can press this button to add more PowerShell debug configurations. One handy configuration to add is **PowerShell: Launch Script**.
+With this configuration, you can specify a specific file with optional arguments that should be launched whenever you press F5 no matter which file is currently active in the editor.
+
+Once the debug configuration is established, you can select which configuration you want to use during a debug session by selecting one from the debug configuration dropdown in the **Debug** view's toolbar.
```
Comment (SteveL-MSFT) [inline docs/learning-powershell/using-vscode.md]: Should be 'drop-down'

File: docs/learning-powershell/using-vscode.md [hunk 2]
```diff
@@ -73,56 +73,91 @@ you will need to add a new variable to your user settings file.
...
+However, workspace debugging allows you to define multiple debug configurations other than just debugging the currently open file.
+For instance, you can add a configurations to:
+
+* Launch Pester tests in the debugger
+* Launch a specific file with arguments in the debugger
+* Launch an interactive session in the debugger
+* Attach the debugger to a PowerShell host process
+
+Follow these steps to create your debug configuration file:
+1. Open the **Debug** view by pressing **Ctrl+Shift+D** (**Cmd+Shift+D** on Mac).
```
Comment (SteveL-MSFT) [inline docs/learning-powershell/using-vscode.md]: Add two spaces at the end of each line to get this formatted as a linebreak

File: docs/learning-powershell/using-vscode.md [hunk 3]
```diff
@@ -73,56 +73,91 @@ you will need to add a new variable to your user settings file.
...
--	Open a file folder (**File->Open Folder**) that contains the PowerShell modules or scripts you have written already and want to debug.
-In this example, we saved the helloworld.ps1 under a directory called "demo".
-Thus we select the "demo" folder and open it in Visual Studio Code.
-
--	Creating the Debug Configuration (launch.json)
-
-  Because some information regarding your scripts is needed for debugger to start executing your script, we need to set up the debug config first.
-  This is one-time process to debug PowerShell scripts under your current folder.
-  In our case, the "demo" folder.
+   When you do this, Visual Studio Code creates a directory and a file **.vscode\launch.json** in the root of your workspace folder.
```
Comment (SteveL-MSFT) [inline docs/learning-powershell/using-vscode.md]: Genereally, it seems file paths are in quotes, so if you put this in quotes, I think the spellchecker will be fine with it

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: docs/learning-powershell/using-vscode.md [hunk 4]
```diff
@@ -16,46 +16,46 @@ Editing with Visual Studio Code
...
 
 **2. Installing PowerShell Extension**
 
--	Launch the Visual Studio Code app by:    
-  *	**Windows**:      typing **code** in your PowerShell session
-  *	**Linux**:        typing **code .** in your terminal
-  *	**macOS**:         typing **code** in your terminal
+-	Launch the Visual Studio Code app by:
+    *	**Windows**:      typing **code** in your PowerShell session
+    *	**Linux**:        typing **code .** in your terminal
```
Comment (SteveL-MSFT) [inline docs/learning-powershell/using-vscode.md]: Perhaps Code required a path on Linux previously, but that doesn't seem the be the case anymore.  just `code` is fine for Linux

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: docs/learning-powershell/using-vscode.md (hunk 2 above)
Comment (rkeithhill) [inline docs/learning-powershell/using-vscode.md]: I've made all the other changes. But I'm not sure I follow this one.  What is the issue?  I have VSCode set up to trim trailing whitespace which is why this probably changed.

**Review by SteveL-MSFT — APPROVED** [review]

**Comment by SteveL-MSFT** [comment]
@rkeithhill looks like you'll need to add `vscode` as a valid word in `.spelling` for in the ` - docs/learning-powershell/using-vscode.md` section

**Pull request merged**